nixos/modules/services/misc/paperless.nix

```diff
@@ -364,9 +364,6 @@ in
         SystemCallFilter = defaultServiceConfig.SystemCallFilter ++ [ "@setuid mbind" ];
         # Needs to serve web page
         PrivateNetwork = false;
-      } // lib.optionalAttrs (cfg.port < 1024) {
-        AmbientCapabilities = [ "CAP_NET_BIND_SERVICE" ];
-        CapabilityBoundingSet = [ "CAP_NET_BIND_SERVICE" ];
```

Comment:
Paperless is actually run without `PrivateUsers`. See line 359.

Reply:
Oops, sorry, you are right. Thanks for the clarification and sorry for taking so long to followup on this.